inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -80,8 +88,29 @@ void FrontEnd::parseInterp(const Model& model, const ie::CNNLayerPtr& layer, con
     VPU_THROW_UNLESS(outputs.size() == 1,
                      "Interp stage with name {} must have only 1 output, "
                      "actually provided {}", layer->name, outputs.size());
+    ie::details::CaselessEq<std::string> cmp;
+    const auto coord = layer->GetParamAsString("coordinate_transformation_mode", "half_pixel");
+    const auto interpMode = layer->GetParamAsString("mode", "linear");
+    InterpolateCoordTransMode coordinateTransMode = InterpolateCoordTransMode::HalfPixel;
+    InterpolateMode mode = InterpolateMode::Linear;
+
+    if (cmp(coord, "asymmetric")) {
+        coordinateTransMode = InterpolateCoordTransMode::Asymmetric;
+    } else if (cmp(coord, "half_pixel")) {
+        coordinateTransMode = InterpolateCoordTransMode::HalfPixel;
+    } else if (cmp(coord, "pytorch_half_pixel")) {
+        coordinateTransMode = InterpolateCoordTransMode::PytorchHalfPixel;
+    } else if (cmp(coord, "tf_half_pixel_for_nn")) {
+        coordinateTransMode = InterpolateCoordTransMode::TfHalfPixelForNn;
+    } else if (cmp(coord, "align_corners")) {
+        coordinateTransMode = InterpolateCoordTransMode::AlignCorners;
+    }
```

Comment:
Should we check that if there is another value we will throw an exception?

Reply:
Yes, right, we should

Reply:
done